tests/h/schemas/group_test.py

```diff
@@ -45,3 +46,23 @@ def test_it_validates_with_valid_name(self):
         })
 
         assert 'name' in appstruct
+
+    def test_it_validates_with_valid_description(self):
+        schema = CreateGroupAPISchema()
+
+        appstruct = schema.validate({
+            'name': 'This Seems Fine',
+            'description': 'This description seems adequate'
+        })
+
+        assert 'description' in appstruct
+
+    def test_it_raises_if_description_too_long(self):
+        schema = CreateGroupAPISchema()
+        with pytest.raises(ValidationError) as exc:
+            schema.validate({
+                'name': 'Name not the Problem',
+                'description': 'o' * (GROUP_DESCRIPTION_MAX_LENGTH + 1)
+            })
+        assert "description:" in str(exc.value)
+        assert "is too long" in str(exc.value)
```

Comment:
I think this can be done with just:

```python
with pytest.raises(ValidationError, match=".*description:.*is too long.*"):
    schema.validate(...)
```

Reply:
I have a slight preference for my version simply because it's easier to read as a human, but I'll swap over if your opinion on this is strong!